### DISCUSSION TITLE
[Attention] {24-11-26}  [업무 분석 및 요건 정의] 업무 분담 件

### DISCUSSION THREAD

**Discussion by cookiecookiekim** [LYNK-LIFE/Groupware#28] [Attention]
### 🔴 중요 공지사항 🔴

```bash
1. [업무 분석 및 요건 정의] 업무 분담 件
- 처리 업무 : (업무 분석 / 요구사항 명세(#27 참고) / 프로토타이핑 / 단위 업무 정의서 / 단위 업무 흐름도)

- 미팅 시간 : 24-11-27(수) 09:40

- 비고 : 시간이 넉넉치 않아, 개발을 빠르게 진행해야 할 것 같습니다. (시행착오를 위해)
[업무 분석 및 요건 정의]는 수(27)요일이나 늦어도 목(28)요일 까지 마무리 하면 좋을 것 같습니다.

기타 의견 있으시거나 시간 조율이 필요하신 경우, 댓글 부탁 드립니다.
```

### Which area of this project could be most improved?

Documentation

### ☑ 확인 결과 체크 ☑

- [X] 김규남
- [x] 김남규
- [x] 윤이정
- [x] 이승현

**Comment by MelodyYD** [comment]
업무 분석 : 법률 관련하여 생길 수 있는 것들을 찾아두는 걸로 이해하긴 했는데, 명확한 건 좀 더 찾아봐야 알 거 같습니다. 내일 한번 이야기해봐요.
요구 사항 명세 : 지금 정해지지 않은 파트에 대해서는 아직 누가 할지 정해지지 않아서, 이건 업무분담하여 내일 최대한 작성하면 좋겠습니다. 그 페이지 개발을 직접 맡지 않더라도 우선은 명세 작성을 위한 분담을 먼저 하는 건 어떨까요?
프로토타이핑 : 초기 구성(프로토타입)을 시각화하는 것을 프로토타이핑이라 하는 거 같아요. 화면 구성 등을 고민해보면 좋을 거 같습니다. 이건 전에... 뭐였죠 그... 쌤이 css 할 때 했던 웬 사이트로라도 구현해보면 좋을 거 같아요. (어려우면 그냥 그림으로 그려도 좋고요.)
단위 업무 정의서, 단위 업무 흐름도 : 명세가 나와야 더 할 수 있을 거 같아요. 우선 명세까지를 빠르게 완료하는 걸로 해요.
